examples/ChatQnA/deploy/k8s_getting_started.md

```diff
@@ -0,0 +1,90 @@
+
+#### Create and Set Namespace
+A Kubernetes namespace is a logical division within a cluster that is used to isolate different environments, teams, or projects, allowing for finer control over resources and access management. To create a namespace called `chatqa`, use:
+```bash
+kubectl create ns chatqa
+```
+When deploying resources (like pods, services, etc.) into your specific namespace, use the `--namespace` flag with `kubectl` commands, or specify the namespace in your resource configuration files.
+
+To deploy a pod in the `chatqa` namespace:
+```bash
+kubectl apply -f your-pod-config.yaml --namespace=chatqa
+```
+If you want to avoid specifying the namespace with every command, you can set the default namespace for your current context:
+```bash
+kubectl config set-context --current --namespace=chatqa
+```
+
+### Using Helm Charts to Deploy
+
+**What is Helm?** Helm is a package manager for Kubernetes, similar to how apt is for Ubuntu. It simplifies deploying and managing Kubernetes applications through Helm charts, which are packages of pre-configured Kubernetes resources.
+
+#### Key Components of a Helm Chart
+
+| Component         |Description                                                                                                                                             |
+| ---               | ---                                                                                                                                                    |
+| `Chart.yaml`      | This file contains metadata about the chart such as name, version, and description.                                                                    |
+| `values.yaml`     | Stores configuration values that can be customized depending on the deployment environment. These values override defaults set in the chart templates. |
+| `deployment.yaml` | Part of the templates directory, this file describes how the Kubernetes resources should be deployed, such as Pods and Services.                       |
+
+**Update Dependencies:**
+
+- A script called **./update_dependency.sh** is provided which is used to update chart dependencies, ensuring all nested charts are at their latest versions.
+- The command `helm dependency update chatqna`  updates the dependencies for the `chatqna` chart based on the versions specified in `Chart.yaml`.
+
+**Helm Install Command:**
+
+- `helm install [RELEASE_NAME] [CHART_NAME]`: This command deploys a Helm chart into your Kubernetes cluster, creating a new release. It is used to set up all the Kubernetes resources specified in the chart and track the version of the deployment.
+
+For more detailed instructions and explanations, you can refer to the [official Helm documentation](https://helm.sh/docs/).
+
+### Using Kubernetes Manifest to Deploy
+Manifest files in YAML format define the Kubernetes resources you want to manage. The main components in a manifest file include:
```

Comment:
Section should be be removed.   Manifests will be removed because they're generated from Helm charts, and they don't catch all the configuration options possible with Helm charts.